tests/Vue/Pages/Home.spec.js

```diff
@@ -64,4 +64,34 @@ describe('Home.vue', () => {
         expect(dialog.isVisible()).toBe(true);
     });
 
+    it('validates that items in textarea input dont exceed the maximum number of ids', async () => {
+        const store = new Vuex.Store();
+
+        const itemsInput = rangeIds(1, MAX_NUM_IDS + 2).join('\n');
```

Comment:
started from 1 because testing locally for Q0 has caused errors for me in the past.

Reply:
Nice! The test looks great, thank you! I think we don't actually have to increment the QIDs, since the sanitization function doesn't deduplicates the IDs in the list, so instead of adding a function we can write something like:

```suggestion
        const itemsInput = Array(MAX_NUM_IDS + 2).fill('Q21');
```